packages/ceramic-common/package.json

```diff
@@ -55,7 +55,8 @@
     "tmp-promise": "^2.0.2",
     "ts-node": "^8.10.2",
     "tsc-watch": "^4.2.9",
-    "typescript-json-schema": "^0.42.0"
+    "typescript-json-schema": "^0.42.0",
+    "dids": "^0.2.1"
```

Comment:
There has been new releases since v0.2 so there might be breaking changes

Reply:
cool, I'll bump it 👍